<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
 background-image : https://home.sophos.com/en-us/medialibrary/Microsites/Home/SecurityCenter/ai-article-pic10.jpg;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backgroundImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/main_background2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color : #fff;
font-size : 42px;
}</string>
    </property>
    <property name="text">
     <string>CPU - SCHEDULING</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{

font-size : 24px;
color : rgb(255, 85, 6);
}</string>
    </property>
    <property name="text">
     <string>SIMULATION PROGRAM</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{

border: 4px dashed #fff;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
border : none;
color : #fff;
font-size : 22px;
}</string>
     </property>
     <property name="text">
      <string>No of Process   : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
border : none;
color : #fff;
font-size : 22px;
}</string>
     </property>
     <property name="text">
      <string>Algorithm         :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>  Select the Scheduling Algorithm</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
background-color : #fff;
font-size : 18px;
}</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="none">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FCFS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Priority-np</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Priorit-p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SJF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SRTF</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nueva Std</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  background-color: rgb(255, 16, 80);
  border-radius : 2px;
  color: white;
  padding: 15px 28px;
  text-align: center;
  font-size: 24px;
}
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
